portfolio/src/main/webapp/script.js

```diff
@@ -74,9 +74,144 @@ function addRandomQuote() {
  */
 /* global google */
 function initMap() {
-  new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: -34.397, lng: 150.644}, zoom: 8});
+  const mapProps = {
+    center: {lat: 19.279619, lng: 166.6499348},
+    zoom: 2,
+
+    // aubergine color scheme
+    styles: [
```

Comment:
This is another example of significant data that's showing up in your JS file. Is it practical to load this from another file?

Unfortunately, I don't know the answer to this myself, but I'm happy to discuss it or help look into it if you'd like.